### PR TITLE
Swap out LabelImage AnswerChoices WB Dropdown

### PR DESCRIPTION
**Pull request by nedredmond**
## Summary:
Swaps out custom dropdown for Wonderblocks dropdown.

This required a litany of changes-- some of the biggest:
 - Move AnswerPill into Marker (also gets rid of warning in console re: calling `findDomNode` in render function)
 - Moving/removing several `onClick` functions so as not to interfere with `opener`'s ClickableState magic-- as a result, analytics functions moved
 - Add logic to WonderBlocks Dropdown to pass to `opener` component when dropdown is open
 - Change how `innerIcon` in marker is rendered (for some reason, refused to be clickable when rendered conditionally)
 - [Also includes changes described in this ADR](https://khanacademy.atlassian.net/l/cp/FHith04E)

And there were some beneficial side-effects:
 - Many styles were consolidated
 - Handling of active/focused markers moved to WB Dropdown
 - Marker & AnswerPill get to share hover/focus states
 - Fixed minor UI issues with Marker being slightly off its mark
 

https://github.com/Khan/perseus/assets/23404711/baa7ebec-326b-468b-b7ce-2af218d1fd85

 Issue: LC-1425
 
 ## Testing strategy:
 Will be thoroughly tested to ensure if still operates the same or better as in prod.

### CODE REVIEW

File: packages/perseus/src/widgets/label-image/marker.tsx
Comment (nedredmond) [inline packages/perseus/src/widgets/label-image/marker.tsx:131]: For some unknown reason, this was not clickable when rendered conditionally. @SonicScrewdriver can attest!

Could be a weird Perseus Icon thing. Decided to hold off switching to WB.

File: packages/perseus/src/widgets/label-image/marker.tsx
Comment (nedredmond) [inline packages/perseus/src/widgets/label-image/marker.tsx:129]: Moved the button logic out of this component. The `opener` function on the dropdown was acting funky and it was unclickable until I moved the clickable component.

File: packages/perseus/src/widgets/label-image/marker.tsx
Comment (nedredmond) [inline packages/perseus/src/widgets/label-image/marker.tsx:162]: Moved this into here to take advantage of the ref (previously, had to do an illegal move and use `findDomNode` in the render function)

File: packages/perseus/src/widgets/label-image/marker.tsx
Comment (nedredmond) [inline packages/perseus/src/widgets/label-image/marker.tsx:94]: Hover styles for initial view-- we enforce this everywhere else

File: packages/perseus/src/widgets/label-image.test.ts
Comment (jeremywiebe) [inline packages/perseus/src/widgets/label-image.test.ts:322]: Phew, were all these tests verifying things that the WB components handle now?

**Review by jeremywiebe — APPROVED** [review]
Whew! That's alot of changes.  😅 

But, the experience looks a bunch better and it's nice to remove more custom code in favour of WB components. Thanks for cleaning this up Ned!

File: packages/perseus/src/widgets/label-image.test.ts
Comment (nedredmond) [reply packages/perseus/src/widgets/label-image.test.ts:322]: This actually just doesn't work, but I meant to un-delete this since I left the non-working code in there (trying to control unnecessary scope creep where I can)

Comment (nedredmond) [reply packages/perseus/src/widgets/label-image.test.ts:322]: It doesn't work in prod either, not just in the PR code

File: packages/perseus/src/widgets/label-image.tsx
Comment (nedredmond) [inline packages/perseus/src/widgets/label-image.tsx]: This can't be exposed because it may include TeX and MD

File: packages/perseus/src/util/sre/tex-to-text.ts
Comment (benchristel) [inline packages/perseus/src/util/sre/tex-to-text.ts:20]: It might be worth clarifying here that liteAdaptor doesn't touch the browser's actual DOM.

**Review by benchristel — COMMENTED** [review]
This is very cool! I like that we're able to reuse MathJax's capabilities for math-to-speech.

One thing that's concerning: the bundle size increased by a couple hundred kb, even though this PR removes more lines than it adds. That smells like MathJax is getting bundled somehow, even though it's only listed as a peer dependency. I think that's worth figuring out before we land this, since we definitely don't want to ship two copies of MathJax in webapp.

**Comment by nedredmond** [comment]
@benchristel MathJax was not previously a part of this repository, so it's added as a peer dep and dev dep, and not actually included when Perseus is imported into webapp.

File: packages/perseus/src/widgets/label-image/answer-pill.tsx
Comment (jeremywiebe) [inline packages/perseus/src/widgets/label-image/answer-pill.tsx:11]: **question**: Can this be moved to the `styles` object below?

**Review by jeremywiebe — COMMENTED** [review]
I was looking through the recent changes and had one nitpicky question.

File: packages/perseus/src/widgets/label-image/answer-pill.tsx
Comment (nedredmond) [reply packages/perseus/src/widgets/label-image/answer-pill.tsx:11]: The styles object doesn't take variables

Comment (nedredmond) [reply packages/perseus/src/widgets/label-image/answer-pill.tsx:11]: Weirdly, it doesn't work when it's there.

**Pull request merged**
